### PR TITLE
Enums in template

### PR DESCRIPTION
**Pull request by fetis**
Small snippet about who to get enums in templates

### CODE REVIEW

File: snippets/enum-in-template.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  CAT,
+  DOLPHIN
+}
+
+...
+@Component({
+  ...
+})
+export class AppComponent {
+  animalsEnum = Animals;
```
Comment (thekiba) [inline snippets/enum-in-template.md]:
```suggestion
  animalsEnum: typeof Animals = Animals;
```

File: snippets/enum-in-template.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  DOLPHIN
+}
+
+@Component({
+  selector: "app-root",
+  template: `<div *ngIf="value === animalsEnum.CAT">meow</div>`,
+})
+export class AppComponent {
+  value: Animals = Animals.CAT;
+  animalsEnum = Animals;
```
Comment (thekiba) [inline snippets/enum-in-template.md]:
```suggestion
  animalsEnum: typeof Animals = Animals;
```

**Review by thekiba — DISMISSED** [review]

File: snippets/enum-in-template.md (hunk 1 above)
Comment (fetis) [reply snippets/enum-in-template.md]: Why it’s better?

Comment (thekiba) [reply snippets/enum-in-template.md]: Strong type checking 🐥

**Review by kirjs — APPROVED** [review]

**Pull request merged**